### PR TITLE
Fix text-ui-test

### PR DESCRIPTION
**Pull request by lowjiayee**
- Change userInput to non-static
- Close scanner before program terminates
- Delete ACTUAL.TXT and Duke.log after every test
- Add event-related test inputs

### CODE REVIEW

File: text-ui-test/runtest.bat [hunk 1]
```diff
@@ -17,3 +17,5 @@ java -jar %jarloc% < ..\..\text-ui-test\input.txt > ..\..\text-ui-test\ACTUAL.TX
 cd ..\..\text-ui-test
 
 FC ACTUAL.TXT EXPECTED.TXT >NUL && ECHO Test passed! || Echo Test failed!
+
+del ACTUAL.TXT Duke.log
```
Comment (benchan911) [inline text-ui-test/runtest.bat:21]: YESSS, Good Job!

File: src/main/java/seedu/duke/Duke.java [hunk 2]
```diff
@@ -65,6 +65,8 @@ public void run() {
                 System.out.println(m.getMessage());
             }
         } while (isNotBye(command));
+
+        ui.close();
```
Comment (benchan911) [inline src/main/java/seedu/duke/Duke.java:69]: Why must we close the scanner? (Just a question to learn)

**Review by benchan911 — APPROVED** [review]
Good to merge

File: src/main/java/seedu/duke/Duke.java (hunk 2 above)
Comment (lowjiayee) [reply src/main/java/seedu/duke/Duke.java:69]: https://www.quora.com/Why-do-we-need-to-close-the-scanner-class-in-java?share=1

Btw, I didn't know why need to close until now 😃

**Pull request merged**